x-pack/plugins/cases/server/client/cases/bulk_get.ts

```diff
@@ -14,52 +14,49 @@ import { pick, partition } from 'lodash';
 import { MAX_BULK_GET_CASES } from '../../../common/constants';
 import type {
   CasesBulkGetResponse,
-  CasesBulkGetResponseCertainFields,
-  CasesBulkGetRequestCertainFields,
-  Case,
+  CasesBulkGetRequest,
   CaseAttributes,
 } from '../../../common/api';
 import {
   CasesBulkGetRequestRt,
-  CasesRt,
+  CasesBulkGetResponseFieldsRt,
   excess,
   throwErrors,
-  getTypeForCertainFieldsFromArray,
-  CaseRt,
+  CasesBulkGetResponseRt,
 } from '../../../common/api';
-import { getTypeProps } from '../../../common/api/runtime_types';
 import { createCaseError } from '../../common/error';
-import { asArray, flattenCaseSavedObject } from '../../common/utils';
+import { flattenCaseSavedObject } from '../../common/utils';
 import type { CasesClientArgs, SOWithErrors } from '../types';
-import { includeFieldsRequiredForAuthentication } from '../../authorization/utils';
 import { Operations } from '../../authorization';
 import type { CaseSavedObjectTransformed } from '../../common/types/case';
 
 type CaseSavedObjectWithErrors = SOWithErrors<CaseAttributes>;
+type BulkGetCase = CasesBulkGetResponse['cases'][number];
 
 /**
  * Retrieves multiple cases by ids.
  */
-export const bulkGet = async <Field extends keyof Case = keyof Case>(
-  params: CasesBulkGetRequestCertainFields<Field>,
+export const bulkGet = async (
+  params: CasesBulkGetRequest,
   clientArgs: CasesClientArgs
-): Promise<CasesBulkGetResponseCertainFields<Field>> => {
+): Promise<CasesBulkGetResponse> => {
   const {
     services: { caseService, attachmentService },
     logger,
     authorization,
   } = clientArgs;
 
   try {
-    const fields = includeFieldsRequiredForAuthentication(asArray(params.fields));
+    const fields = Object.keys(CasesBulkGetResponseFieldsRt.props).filter(
```

Comment:
Do we need to omit the `version` and `id` here too?

Reply:
Good point.